### PR TITLE
`azurerm_app_service_certificate_binding` - rework for removal of thumbprint from service

### PR DESCRIPTION
**Pull request by jackofallops**
Related to #13353

### CODE REVIEW

File: internal/services/web/app_service_certificate_binding_resource.go [hunk 1]
```diff
@@ -126,7 +123,7 @@ func resourceAppServiceCertificateBindingCreate(d *pluginsdk.ResourceData, meta
 
 	props := binding.HostNameBindingProperties
 	if props != nil {
-		if props.Thumbprint != nil && *props.Thumbprint == *thumbprint {
+		if props.SslState != "" || props.VirtualIP != nil {
```
Comment (tombuildsstuff) [inline internal/services/web/app_service_certificate_binding_resource.go:126]: can't SSL State be disabled?

**Review by tombuildsstuff — APPROVED** [review]
one minor comment but this otherwise LGTM 👍

File: internal/services/web/app_service_certificate_binding_resource.go (hunk 1 above)
Comment (jackofallops) [reply internal/services/web/app_service_certificate_binding_resource.go:126]: Yes, but it seems that only un-configured bindings are empty for this property. If it has a setting, it is in some way being managed and I think should be imported.

**Comment by jackofallops** [comment]
Tests looking good:
image

**Review by tombuildsstuff — APPROVED** [review]

**Pull request merged**